pkg/components/openebs-operator/component.go

```diff
@@ -79,6 +79,10 @@ ndmOperator:
 
 {{- if and .NDMSelectorLabel .NDMSelectorValue }}
 webhook:
+  # OpenEBS by default ships the failurePolicy as 'Fail', however this causes sometimes to unexpectedly
+  # hit a known bug in OpenEBS https://github.com/openebs/openebs/issues/3046.
+  # One of the workarounds suggested is to change the failurePolicy to "Ignore".
+  failurePolicy: 'Ignore'
```

Comment:
`if` condition must be moved below, otherwise it will be ignored only if selectors are specified.

Reply:
Oops. Will fix.